pulp_ansible/app/serializers.py

```diff
@@ -142,7 +142,8 @@ def _validate_url(url):
 
         data = super().validate(data)
 
-        _validate_url(data["url"])
+        if data.get("url"):
```

Comment:
what motivated this change?

Reply:
when doing a PATCH it was breaking because no URL was being sent